doc/designs/lms.md

```diff
@@ -0,0 +1,194 @@
+Leighton-Micali Signature (LMS)
+===============================
+
+Introduction
+------------
+
+Existing OpenSSL digital signature algorithms such as ECDSA and EDDSA
+have the potential to be broken using large scale quantum computers.
+
+The NSA have listed LMS as a quantum-resistant scheme for software and
+firmware signing for National Security Systems [3].
+LMS is a Stateful Hash Based Signature (HBS) scheme that depends only on the
```

Comment:
Generally "hash-based" is hyphenated, both in prose and when in title case (i.e. Hash-Based Signatures)